### PR TITLE
Update configurations requests

### PR DESCRIPTION
**Pull request by gabrieltron**

### CODE REVIEW

**Comment by gabrieltron** [comment]
[[Config] Update Postman and docs](https://trello.com/c/TwLHdTJZ/1153-config-update-postman-and-docs)

File: Leaf API.postman_collection.json [hunk 1]
```diff
@@ -1834,7 +1835,7 @@
 										],
 										"body": {
 											"mode": "raw",
-											"raw": "{\n    \"operationsImageCreation\": true,\n    \"operationsSummaryCreation\": true,\n    \"fieldsAutoSync\" : true\n}"
+											"raw": "{\n    \"operationsImageCreation\": true,\n    \"fieldsAutoSync\" : true\n}"
```
Comment (fabyan-alexander) [inline Leaf API.postman_collection.json:1838]: Should we include all the properties here?

File: Leaf API.postman_collection.json [hunk 2]
```diff
@@ -1734,7 +1735,7 @@
 										],
 										"body": {
 											"mode": "raw",
-											"raw": "{\n    \"operationsImageCreation\": false,\n    \"operationsSummaryCreation\": false\n}"
+											"raw": "{\n    \"operationsImageCreation\": false\n}"
```
Comment (fabyan-alexander) [inline Leaf API.postman_collection.json:1738]: Should we include all the properties here?

**Review by fabyan-alexander — CHANGES_REQUESTED** [review]
Nice update! Just a few questions

File: Leaf API.postman_collection.json (hunk 1 above)
Comment (gabrieltron) [reply Leaf API.postman_collection.json]: This is related to the PATCH, so I don't think it's obligatory to include all properties, but there's certainly no harm on putting it there. Done in 1005b0350abaa5483615d613b4390ecc3662bd8c

File: Leaf API.postman_collection.json (hunk 2 above)
Comment (gabrieltron) [reply Leaf API.postman_collection.json]: Done in 1005b0350abaa5483615d613b4390ecc3662bd8c

**Pull request closed without merging**